render_config.sh

```diff
@@ -18,23 +13,17 @@ fi
 
 DSDE_TOOLBOX_DOCKER_IMAGE=broadinstitute/dsde-toolbox:consul-0.20.0
 
-SERVICE_ACCOUNT_CREDS=`docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN ${DSDE_TOOLBOX_DOCKER_IMAGE} \
-    vault read -format=json ${FIRECLOUD_ACCOUNT_VAULT_PATH} | \
-    jq .data`
+if type vault > /dev/null; then
```

Comment:
I don't have `vault` installed currently, and this line fails.
`./render_config.sh: line 16: type: vault: not found`

Reply:
yeah, I say stick with the docker image, to prevent issues with different versions of the `vault` binary, or aliases, or anything like that.